extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -83,14 +83,7 @@ private function getInsightModule( $type, $config ) {
 		 */
 		$insightPages['type'] = $type;
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type . '-title' )->text();
-		$insightPages['description'] =  wfMessage( 'communitypage-' . $type . '-description' )->text();
-
-		// TODO: to be removed in the scope of: https://wikia-inc.atlassian.net/browse/WW-83
-		$insightPages['mediumtitle'] = wfMessage( 'communitypage-' . $type . '-mediumtitle' )->text();
-		$insightPages['hightitle'] = wfMessage( 'communitypage-' . $type . '-hightitle' )->text();
-		$insightPages['mediumdescription'] =  wfMessage( 'communitypage-' . $type . '-mediumdescription' )->text();
-		$insightPages['highdescription'] =  wfMessage( 'communitypage-' . $type . '-highdescription' )->text();
-
+		$insightPages['description'] =  $type === 'wantedpages' ? wfMessage( 'communitypage-wantedpages-description' )->text() : null;
```

Comment:
Per our coding conventions this line is too long.
https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#line-length